base/usage/v1alpha/usage.proto

```diff
@@ -95,22 +96,25 @@ message MgmtUsageData {
 message ConnectorUsageData {
   // Per user usage data in the connector service
   message UserUsageData {
+    // Per execute usage metadata
+    message ConnectorExecuteData {
+      // UID for the executed connector
+      string connector_uid = 1 [(google.api.field_behavior) = REQUIRED];
+      // UID for the trigger log
+      string execute_uid = 2 [(google.api.field_behavior) = REQUIRED];
+      // Timestamp for the execution
+      google.protobuf.Timestamp execute_time = 3 [(google.api.field_behavior) = REQUIRED];
+      // Definition ID of the connector
+      string connector_definition_id = 4 [(google.api.field_behavior) = REQUIRED];
```

Comment:
@heiruwu and @donch1989, should we use the connector_definition_id or connector_definition_uid?

Reply:
I think `connector_definition_uid ` would be better. Because sometimes we may change the connector_definition_id.